### DISCUSSION TITLE
Architectural Review

### DISCUSSION THREAD

**Discussion by xeolabs** [xeokit/sdk#16] [Announcements]
API is now pretty stable and documentation almost complete (for the core of the SDK).

What would be great now would be:

* opinions on names of things - anything that should be renamed for a better "user mental model"?
* any code smells that must be factored out?
* architectural changes?
* new features that look doable on the new architecture?
* any new tech we should use to manege the project?

Don't worry about any erros in the API docs - they're still work in progress. Although, any comments on how those might be improved are always great.

* API Docs entry point: https://xeokit.github.io/sdk/docs/pages/OVERVIEW.html

Also note that no npm modules have been published yet.

**Comment by DhivinX** [comment]
@xeolabs suitable for discussion

Edit: 
You might want to move this thread to discussion tab ;)
